www/www-deploy.yml

```diff
@@ -99,6 +99,8 @@
     # about
     - match: "~/site/about/?$"
```

Comment:
Not entirely clear on the syntax here - do I need the `(?<link>.*)$` bit here to default to redirecting all non-specified about links to the new page?

Reply:
- `~/site/about/.*$` if defined last in the list will pick up any pattern that did not match one of the previous regexps
-`~/site/about/(<?link>.*)$` is the same but allows to store the matched pattern e.g. to reuse it and construct the  redirection URL

Where do you want to redirect anything that is not matched?

Reply:
I want everything that doesn't have a specific redirect to just go to /about